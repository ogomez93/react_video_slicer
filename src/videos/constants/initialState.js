import { DEFAULT_VIDEO_URL } from './index';

export default [
  {
    duration: 0,
    clips: [],
    loading: true,
    paused: true,
    videoUrl: DEFAULT_VIDEO_URL
  }
];
