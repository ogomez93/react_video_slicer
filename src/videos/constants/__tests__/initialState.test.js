import initialState from '../initialState';

describe('Videos / Constants / initialState', () => {
  it('should match the snapshot', () =>
    expect(initialState).toMatchSnapshot());
});
