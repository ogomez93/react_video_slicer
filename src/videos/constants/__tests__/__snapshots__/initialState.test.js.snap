// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Videos / Constants / initialState should match the snapshot 1`] = `
Array [
  Object {
    "clips": Array [],
    "duration": 0,
    "loading": true,
    "paused": true,
    "videoUrl": "https://download.blender.org/durian/trailer/sintel_trailer-480p.mp4",
  },
]
`;
